### ISSUE TITLE
Check and fix broken links

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#15]
Broken links are links which do not behave as expected when used. In most cases, you will want to fix broken links.

Links can be broken in many ways, and it is possible for a link to appear broken for some people and not for others. For example, some links may only work if the person using them is in a specific country, or has access to a protected network link a company intranet.

Links which point to a missing website, a missing page, or which display an error message are shown here.
